psi4/driver/driver_cbs.py

```diff
@@ -1307,46 +1092,13 @@ def _contract_scheme_orders(needdict, datakey='f_energy'):
     return largs
 
 
-##  Aliases  ##
-complete_basis_set = cbs
-
-
-def _cbs_wrapper_methods(**kwargs):
-    """ A helper function for the driver to enumerate methods used in the
-    stages of a cbs calculation.
-
-    Parameters
-    ----------
-    kwargs : dict
-        kwargs containing cbs specification either in the ``cbs_metadata``
-        format, or in separate keywords (``scf_wfn``, ``corl_wfn`` etc.).
-
-    Returns
-    -------
-    list
-        List containing method name for each active stage.
-    """
-
-    cbs_methods = []
-    if "cbs_metadata" in kwargs:
-        for item in kwargs["cbs_metadata"]:
-            cbs_methods.append(item.get("wfn"))
-    else:
-        cbs_method_kwargs = ['scf_wfn', 'corl_wfn', 'delta_wfn']
-        cbs_method_kwargs += [f'delta{x}_wfn' for x in range(2, 6)]
-        for method in cbs_method_kwargs:
-            if method in kwargs:
-                cbs_methods.append(kwargs[method])
-    return cbs_methods
-
-
-def _parse_cbs_gufunc_string(method_name):
+def _parse_cbs_gufunc_string(method_name: str):
```

Comment:
I'd argue this function and `cbs_text_parser` might belong to `task_base.py`, as that's where processing of the method name is done. Not insisting, just food for thought.